providers/flagd/pkg/constant/reason.go

```diff
@@ -0,0 +1,5 @@
+package constant
+
+const (
+	ReasonCached = "CACHED"
```

Comment:
Do you think it's worth adding this `reason` to the spec? Or just to the SDKs?

I'm not sure it's worth the effort, but I'm interested in your opinion.

Reply:
Yeah, I think it makes sense.

Reply:
+1 from me, I think it makes sense to define the behaviour to avoid discrepancies between implementations.

Reply:
Created an issue for this: https://github.com/open-feature/spec/issues/165